cmake/cmake_celix/BundlePackaging.cmake

```diff
@@ -18,6 +18,9 @@
 
 
 set(CELIX_NO_POSTFIX_BUILD_TYPES RelWithDebInfo Release CACHE STRING "The build type used for creating bundle without a build type postfix.")
+set(CELIX_DEFAULT_JAR_COMMAND_ARGUMENTS -cfm0 CACHE STRING "Default no compression is applied")
```

Comment:
Is the intention that users can change/override these settings? In that case I’d leave out the `DEFAULT_` part.

Reply:
Just curious why we made that change?

Reply:
Quicker startup containers

Reply:
I see. A conan package option will be added for this later.